EvalData/models/pairwise_assessment_document.py

```diff
@@ -1090,3 +1091,1067 @@ def completed_results_for_user_and_campaign(cls, user, campaign):
         ).values_list('item_id', flat=True)
 
         return len(set(results))
+    
+    
+@AnnotationTaskRegistry.register
+class PairwiseAssessmentDocumentESATask(BaseMetadata):

```

Comment:
Do we need to copy-paste the entire class or would it be enough to inherit from it? If inheritance is an alternative, I'd prefer it over copy-pasting as this is quite a lot of code.

Reply:
If we use inheritance, it will be also clear which methods are specific to or required changes in the new class.

Reply:
Change to use a combination of abstract classes and inheritance.